### PR TITLE
EN-3679 List all IAM Users Lego

### PR DESCRIPTION
**Pull request by shloka-bhalgat-unskript**
## Description
Added  List all IAM Users Action.

### Testing

Screenshot 2022-12-09 at 4 12 47 PM

### Checklist:
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository.

### CODE REVIEW

File: AWS/legos/aws_list_all_iam_users/aws_list_all_iam_users.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        return
+    pprint.pprint(output)
+
+def aws_list_all_iam_users(handle) -> List:
+    """aws_list_all_iam_users lists all the IAM users
+
+        :type handle: object
+        :param handle: Object returned from Task Validate
+        
+        :rtype: Result List of result
```
Comment (jayasimha-raghavan-unskript) [inline AWS/legos/aws_list_all_iam_users/aws_list_all_iam_users.py:23]: You meant Result List of Regions?

Comment (jayasimha-raghavan-unskript) [reply AWS/legos/aws_list_all_iam_users/aws_list_all_iam_users.py:23]: or users?

**Review by jayasimha-raghavan-unskript — APPROVED** [review]
Please take care of my comments.

File: AWS/legos/aws_list_all_iam_users/aws_list_all_iam_users.py (hunk 1 above)
Comment (shloka-bhalgat-unskript) [reply AWS/legos/aws_list_all_iam_users/aws_list_all_iam_users.py]: Of users. Made the changes, please check

**Pull request merged**